src/process/process.rs

```diff
@@ -164,10 +175,8 @@ impl<'a> Process<'a> {
             // Set kernel break to beginning of callback buffer
             kernel_memory_break =
                 kernel_memory_break.offset(-(callback_offset as isize));
-            let callback_buf = mem::transmute(Slice {
-                data: kernel_memory_break as *const Option<Callback>,
```

Comment:
In other words, it doesn't really matter what the size of the pointer was because `trasmute` will change the type argument of `Slice` to whatever. But the type of `callback_buf` was/is interred to be `&mut [Callback]` because of it's use in `RingBuffer<'a, Callback>::new`.